### DISCUSSION TITLE
Add admin option to configure branch status url

### DISCUSSION THREAD

**Discussion by viceice** [renovatebot/renovate#22208] [Suggest an Idea]
**What would you like Renovate to be able to do?**

I like to configure the url for renovate branch status objects, so i can point the user to a specific url.
I like to use some templating, to add the repo / namespace and maybe other later.

**Did you already have any implementation ideas?**

Sure, add a new option `branchStatusUrl` which can use handlebar templates.

ref #10714

**Comment by rarkins** [comment]
Part of `productLinks`?

**Comment by viceice** [comment]
> Part of `productLinks`?

sure, sounds good
